data/src/main/kotlin/com/antosik/benchproject/data/movies/common/database/dao/MovieDao.kt

```diff
@@ -11,13 +11,17 @@ import com.antosik.benchproject.data.movie.details.entity.database.MovieDetailsE
 import com.antosik.benchproject.data.movie.details.entity.database.relations.MovieDetailsGenresCrossRef
 import com.antosik.benchproject.data.movie.details.entity.database.relations.MovieDetailsWithGenres
 import com.antosik.benchproject.data.popular.movies.entity.database.MovieEntity
+import kotlinx.coroutines.flow.Flow
 
 @Dao
 internal interface MovieDao {
 
     @Query("SELECT * FROM MovieEntity")
     suspend fun getMovies(): List<MovieEntity>
 
+    @Query("SELECT * FROM MovieEntity WHERE isFavorite = 1")
+    fun getFavoriteMovies(): Flow<List<MovieEntity>>
```

Comment:
here also, but it is nothing serious for me :D just favouriteMovies will be enough, usage of java style get can be omited

Reply:
Same like here 😄 https://github.com/Nataniel-Antosik/Bench_Project/pull/182#discussion_r1284193272